src/main/java/bio/terra/service/filedata/google/firestore/FireStoreUtils.java

```diff
@@ -66,21 +70,21 @@ RuntimeException handleExecutionException(Throwable ex, String op) {
             // TODO: in general, log + rethrow is bad form. For now, I want to make sure we see these in
             //  the log as they happen. Once we are comfortable that retry is working properly, we can
             //  rely on the Stairway debug logging as needed.
-            String msg = "FirestoreUtils.handleExecutionException - Retrying aborted exception: " + aex;
+            String msg = "[" + op + "] FirestoreUtils.handleExecutionException - Retrying aborted exception: " + aex;
             logger.info(msg);
             return new FileSystemAbortTransactionException(msg, aex);
         }
         if (throwable instanceof FirestoreException) {
             FirestoreException fex = (FirestoreException) throwable;
-            String msg = "FirestoreUtils.handleExecutionException - Retrying firestore exception: " + fex;
+            String msg = "[" + op + "] FirestoreUtils.handleExecutionException - Retrying firestore exception: " + fex;
```

Comment:
You could use `{}` here too instead of string concatenation. I don't have a preference one way or the other but it's odd to see it done inconsistently in a single method.

Reply:
Can you use that when defining a regular string? I thought we could only do that for logger messages.

Reply:
Oh, I see, the message is used for both the logger and the exception. No, in that case I don't think there's a better way.